File: code/crawllers/douban-python-crawller/entity/Movie.py
class Movie:
    def __init__(self, _id, name, _type, language, ranking,hot):
        self.id = _id
        self.name = name
        self.type = _type
        self.language = language
        self.ranking = ranking
        self.hot = hot

    def __str__(self):
        result = self.id+" "+self.name+" "+ self.type+" "+self.language+" "+self.ranking+" "+self.hot
        return result

File: code/backend/douban-service/src/test/java/com/ilife/douban/entity/Movie.java
package com.ilife.douban.entity;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import javax.persistence.*;

@Entity
@Table(name = "MOVIE")
@Data
@AllArgsConstructor
@NoArgsConstructor
public class Movie {
    @Id
    @Column(name = "ID")
    private String id;
    @Column(name = "NAME")
    private String name;
    @Column(name = "TYPE")
    private String type;
    @Column(name = "LANGUAGE")
    private String language;
    @Column(name = "RANKING")
    private float ranking;
    @Column(name = "HOT")
    private Integer hot;
}